engine/access/rpc/backend/backend_stream_transactions.go

```
@@ -30,55 +33,132 @@ type backendSubscribeTransactions struct {
+// Monitoring starts from the specified block height.
+// If the block height cannot be determined or an error occurs during subscription creation, a failed subscription is returned.
+func (b *backendSubscribeTransactions) SubscribeTransactionStatusesFromStartHeight(
+	ctx context.Context,
+	txID flow.Identifier,
+	startHeight uint64,
+	requiredEventEncodingVersion entities.EventEncodingVersion,
+) subscription.Subscription {
+	return b.createSubscription(ctx, txID, flow.ZeroID, startHeight, flow.ZeroID, requiredEventEncodingVersion, false)
+}
+
+// SubscribeTransactionStatusesFromStartBlockID subscribes to the status updates of a transaction.
+// Monitoring starts from the specified block ID.
+// If the block ID cannot be determined or an error occurs during subscription creation, a failed subscription is returned.
+func (b *backendSubscribeTransactions) SubscribeTransactionStatusesFromStartBlockID(
+	ctx context.Context,
+	txID flow.Identifier,
+	startBlockID flow.Identifier,
+	requiredEventEncodingVersion entities.EventEncodingVersion,
+) subscription.Subscription {
+	return b.createSubscription(ctx, txID, startBlockID, 0, flow.ZeroID, requiredEventEncodingVersion, false)
+}
+
+// SubscribeTransactionStatusesFromLatest subscribes to the status updates of a transaction.
+// Monitoring starts from the latest block.
+// If the block cannot be retrieved or an error occurs during subscription creation, a failed subscription is returned.
+func (b *backendSubscribeTransactions) SubscribeTransactionStatusesFromLatest(
+	ctx context.Context,
+	txID flow.Identifier,
+	requiredEventEncodingVersion entities.EventEncodingVersion,
+) subscription.Subscription {
+	header, err := b.txLocalDataProvider.state.Sealed().Head()
 	if err != nil {
-		return subscription.NewFailedSubscription(err, "could not get start height")
+		b.log.Error().Err(err).Msg("failed to retrieve latest block")
```

This should actually throw an exception since the node must have the current sealed block in storage.
e.g.
```golang
irrecoverable.Throw(ctx, err)
```